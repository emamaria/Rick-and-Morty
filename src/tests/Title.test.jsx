describe('test <Title/>', () => {
    

    test('should match the snapshot', () => {
        console.log("gola")
    })
});
