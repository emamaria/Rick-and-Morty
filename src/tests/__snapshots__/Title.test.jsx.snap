// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <Title/> should match the snapshot 1`] = `
<div>
  <h1>
    Rick And Morty
  </h1>
</div>
`;
